src/validation/patient.validate.ts

```diff
@@ -0,0 +1,38 @@
+import * as yup from 'yup';
+import { useTranslation } from 'react-i18next';
+
+const NAME_PATTERN = /^([A-Z][a-z]{1,11})/;
+
+export function patientSchema() {
+  const { t } = useTranslation();
+
+  const createPatientCardSchema = yup.object().shape({
+    firstName: yup
+      .string()
+      .required(t('Error.firstNameRequired') ?? '')
+      .min(2, t('Error.firstNameRequired') ?? '')
+      .matches(NAME_PATTERN, t('Error.firstNameRequired') ?? ''),
+    lastName: yup
+      .string()
+      .required(t('Error.lastNameRequired') ?? '')
```

Comment:
why we need ?? condition?

Reply:
I still dont understand, question is what ?? do in this situation? 
I see this like - if we have translation Error.lastNameRequired, we show it, if no, we don't show nothing:
1) it will never go to the second condition if I read it right
2) in logic point of view its not right

Reply:
resolved